### DISCUSSION TITLE
Unidentified Annotations

### DISCUSSION THREAD

**Discussion by konashevich** [sumatrapdfreader/sumatrapdf#3313] [General]
Hi
I realized that when I add a lot of annotations on a page then it is not easy to identify them. First of all when I click on an annotation with a comment there is now way to see this commnet (for example, in Chrome browser the annotation-comment pops up as a small window next to the colored text once you hower your mouse on it). I would expect something like that - hover mouse and/or [dobule] click (click is also needed as it will work with touch screens).

The only option to see comments is to open that additional window - left click / Edit annotations. 
But then comes another problem -  if you added dozens of annotaions on a page you basically see the list of Annotaitons in the Edit window. And the only way to match them with the pdf windown is to acount them because when you click on an Annotation in Edit window it does not highlight it anyhow in the PDF window. Thus you don't know which comment belongs to which annotation in the PDF widndow.

**Comment by GitHubRulesOK** [comment]
the annotation when 1st introduced auto copied highlights but  if I recollect more users that did not want it automatic so was changed to optional by paste 
![image](https://user-images.githubusercontent.com/12754913/222007381-5a6b0527-6477-4099-a3ee-69e190b6d034.png)
![image](https://user-images.githubusercontent.com/12754913/222007969-835a7dd6-cda3-47da-bed4-642cebddeb1c.png)

context (hold screen or right click) has go to entry
![image](https://user-images.githubusercontent.com/12754913/222007659-198699db-7d9d-4741-a32e-7e729f3e2eef.png)

**Comment by GJoe2** [comment]
I think there is an issue when you have an annotation and a underline text very closed each other.
Annotation :
![image](https://user-images.githubusercontent.com/71229258/222586024-b71c27cd-1253-47b9-ae83-28312d5cf90c.png)

Underline text:
![image](https://user-images.githubusercontent.com/71229258/222585784-d7c4ed5a-6193-462a-83bd-270c06085832.png)


In the screenshot you can see there is an annotation (T=C=w*h*fc) but somehow the highlighted text (V=w(1-w)h**2*t*fc/a) overwrite it and it only shows the author (user)
![image](https://user-images.githubusercontent.com/71229258/222581916-f0569837-1a1d-4216-963b-be540de774b4.png)

**Reply by GJoe2** [reply]
I believe the issue is cause in how annotation boxes are made
![image](https://user-images.githubusercontent.com/71229258/222587910-dc445924-a75f-4d58-a522-bbd6cb775d72.png)

I think the box is simple enough to make just one box on whatever entity, which may cause overlap as you can see in the picture. 

In other pdf viewer they optimize it to just box the entity where is needed, in the picture below it generates two boxes
![image](https://user-images.githubusercontent.com/71229258/222588682-90aae29c-8815-48f7-89df-75b850ad7faf.png)

**Reply by GJoe2** [reply]
I believe the issue is cause in how annotation boxes are made
![image](https://user-images.githubusercontent.com/71229258/222587910-dc445924-a75f-4d58-a522-bbd6cb775d72.png)

I think the box is simple enough to make just one box on whatever entity, which may cause overlap as you can see in the picture. 

In other pdf viewer they optimize it to just box the entity where is needed, in the picture below it generates two boxes
![image](https://user-images.githubusercontent.com/71229258/222588682-90aae29c-8815-48f7-89df-75b850ad7faf.png)
